modules/prebid/ortb2blocking/module_test.go

```diff
@@ -594,14 +599,698 @@ func TestHandleBidderRequestHook(t *testing.T) {
 	}
 }
 
-type numeric interface {
-	openrtb2.BannerAdType | adcom1.CreativeAttribute
-}
+func TestHandleRawBidderResponseHook(t *testing.T) {
+	testCases := []struct {
+		description        string
+		payload            hookstage.RawBidderResponsePayload
+		config             json.RawMessage
+		moduleCtx          hookstage.ModuleContext
+		expectedBids       []*adapters.TypedBid
+		expectedHookResult hookstage.HookResult[hookstage.RawBidderResponsePayload]
+		expectedError      error
+	}{
+		{
+			description: "Payload not changed when empty account config and empty module contexts are provided. Analytic tags have successful records",
+			payload: hookstage.RawBidderResponsePayload{Bidder: bidder, Bids: []*adapters.TypedBid{
+				{
+					Bid: &openrtb2.Bid{ADomain: []string{"foo"}, ImpID: impID1},
+				},
+			}},
+			expectedBids: []*adapters.TypedBid{
+				{
+					Bid: &openrtb2.Bid{ADomain: []string{"foo"}, ImpID: impID1},
+				},
+			},
+			expectedHookResult: hookstage.HookResult[hookstage.RawBidderResponsePayload]{
+				AnalyticsTags: hookanalytics.Analytics{
+					Activities: []hookanalytics.Activity{
+						{
+							Name:   enforceBlockingTag,
+							Status: hookanalytics.ActivityStatusSuccess,
+							Results: []hookanalytics.Result{
+								{
+									Status:    hookanalytics.ResultStatusAllow,
+									AppliedTo: hookanalytics.AppliedTo{Bidders: []string{bidder}, ImpIds: []string{impID1}},
+								},
```

Comment:
Sorry, one more thing, I overlooked it before. `Bidders: []string{bidder}` in spec bidder is a string, not `[]string`

Reply:
Thank you, updated